src/main/java/chronosacaria/mcdw/api/util/InventoryHelper.java

```diff
@@ -15,16 +16,7 @@ public class InventoryHelper {
     }
 
     public static boolean mcdw$hasItem(PlayerEntity playerEntity, Item item, int count) {
-        PlayerInventory playerInventory = playerEntity.getInventory();
-        for (int slotID = 0; slotID < playerInventory.size(); slotID++) {
-            ItemStack currentStack = playerInventory.getStack(slotID);
-            if (currentStack.getItem() == item) {
-                count -= currentStack.getCount();
-                if (count <= 0)
-                    return true;
-            }
-        }
-        return false;
+        return mcdw$countItem(playerEntity, item) >= count;
```

Comment:
Big facepalm on my part. Good catch!

Reply:
The Chain Reaction thing is almost certainly mismatched parameters, but I'll trace it in a bit to make sure.